packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -185,6 +184,16 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
             return;
```

Comment:
We should remember a loading promise here and return it, that clients await properly.
It would be also good to add a check against cycles, then it should just `return`. It could be done by passing a set of visited plugins.